lib/simple_sqs/processor.rb

```diff
@@ -6,26 +6,26 @@
 require 'logger'
 
 class SimpleSqs::Processor
-  def process_sqs_message sqs_message
+  def process_sqs_message json_message_body, sqs_message = nil
```

Comment:
@shosti  `process_sqs_message` might be used by outsiders. we use it directly in a couple of our spout specs.